### PR TITLE
Notifications test

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: src/components/Notifications/NotificationItem/__tests__/container.spec.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+store.dispatch = jest.fn();
+
+describe('NotificationItem container', () => {
+  const wrapper = shallow(<NotificationItem store={store} notification={{}} />);
+  const container = wrapper.dive();
+  const instance = container.instance();
+
+  it('should render NotificationItem', () => {
+    expect(wrapper).toMatchSnapshot();
```
Comment (plzen) [inline src/components/Notifications/NotificationItem/__tests__/container.spec.js]: Use container for snapshot

File: src/components/Notifications/NotificationItem/__tests__/container.spec.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  });
+
+  describe('should run timers on NotificationItem', () => {
+    it('componentDidMount()', () => {
+      jest.useFakeTimers();
```
Comment (plzen) [inline src/components/Notifications/NotificationItem/__tests__/container.spec.js]: Test just this.timer() call

Comment (plzen) [inline src/components/Notifications/NotificationItem/__tests__/container.spec.js]: Move this to beforeEach

Comment (plzen) [reply src/components/Notifications/NotificationItem/__tests__/container.spec.js]: Add setState test

File: src/components/Notifications/NotificationItem/__tests__/container.spec.js (hunk 1 above)
Comment (eugene-chr) [reply src/components/Notifications/NotificationItem/__tests__/container.spec.js]: done

File: src/components/Notifications/NotificationItem/__tests__/container.spec.js (hunk 2 above)
Comment (eugene-chr) [reply src/components/Notifications/NotificationItem/__tests__/container.spec.js]: done

Comment (eugene-chr) [reply src/components/Notifications/NotificationItem/__tests__/container.spec.js]: doen

**Pull request merged**